cg/constants/gene_panel.py

```diff
@@ -42,9 +42,7 @@ class GenePanelMasterList(StrEnum):
 
     @classmethod
     def get_panel_names(cls, panels=None) -> list[str]:
-        """Return requested panel names from Master list, or all panels if none are specified."""
+        """Return requested panel names from the Master list, or all panels if none are specified."""
         return (
-            [panel_name.value for panel_name in panels]
-            if panels
-            else list(map(lambda panel: panel.value, cls))
+            [panel_name for panel_name in panels] if panels else list(map(lambda panel: panel, cls))
```

Comment:
is the `map` needed here? Seems as if the lambda expression doesn't do anything anymore.

Reply:
Fixed